dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/TracedEncoding.java

```diff
@@ -94,11 +98,15 @@ static String toString(TypeMarker<?> typeMarker) {
 
         @Override
         public void serialize(T value, OutputStream output) throws IOException {
-            Tracer.fastStartSpan(operation);
-            try {
+            if (log.isDebugEnabled()) {
```

Comment:
what's the thinking here and below?

Reply:
We can quickly turn this on at runtime if we need to, but it’s not useful in most scenarios, nor does it make sense to build our own tracing logger configuration point.